### PR TITLE
peerconnection SetRemoteDescription failed

### PR DESCRIPTION
**Pull request by AlexWoo**
#### Description

remote end not support and set first media block inactive, first media block has no fingerprint

SDP as follow:

    v=0
    o=- 143087887 1561022767 IN IP4 150.138.238.87
    s=VideoRoom 404986692241682
    t=0 0
    a=group:BUNDLE audio
    a=msid-semantic: WMS 2867270241552712
    m=video 0 UDP/TLS/RTP/SAVPF 0
    c=IN IP4 150.138.238.87
    a=inactive
    m=audio 9 UDP/TLS/RTP/SAVPF 111
    c=IN IP4 150.138.238.87
    a=recvonly
    a=mid:audio
    a=rtcp-mux
    a=ice-ufrag:AS/w
    a=ice-pwd:9NOgoAOMALYu/LOpA1iqg/
    a=ice-options:trickle
    a=fingerprint:sha-256 D2:B9:31:8F:DF:24:D8:0E:ED:D2:EF:25:9E:AF:6F:B8:34:AE:53:9C:E6:F3:8F:F2:64:15:FA:E8:7F:53:2D:38
    a=setup:active
    a=rtpmap:111 opus/48000/2
    a=candidate:1 1 udp 2013266431 150.138.238.87 46492 typ host
    a=candidate:2 1 udp 2013266431 27.221.118.139 41479 typ host
    a=candidate:3 1 udp 2013266431 223.99.230.212 45043 typ host
    a=candidate:4 1 udp 2013266431 10.42.0.0 45977 typ host
    a=candidate:5 1 udp 2013266431 10.42.0.1 37692 typ host
    a=end-of-candidates
    m=application 0 DTLS/SCTP 0
    c=IN IP4 150.138.238.87
    a=inactive

#### Reference issue
Fixes #...

### CODE REVIEW

File: peerconnection.go [hunk 1]
```diff
@@ -850,6 +850,20 @@ func (pc *PeerConnection) LocalDescription() *SessionDescription {
 	return pc.currentLocalDescription
 }
 
+func mediaClose(m *sdp.MediaDescription) bool {
```
Comment (Sean-Der) [inline peerconnection.go]: can you rename this to `mediaIsClosed`, thanks!

**Comment by Sean-Der** [comment]
Great fix, thank you so much @AlexWoo 

If you have time would you mind adding a test to make sure we don't break this in the future :) If not I can write it! I would move this all into a dedicated function `getFingerprint` or something like that so we can write a nice little unit test for it.

thanks so much for the contribution, and thank you for using Pion :)

**Comment by AlexWoo** [comment]
> Great fix, thank you so much @AlexWoo
> 
> If you have time would you mind adding a test to make sure we don't break this in the future :) If not I can write it! I would move this all into a dedicated function `getFingerprint` or something like that so we can write a nice little unit test for it.
> 
> thanks so much for the contribution, and thank you for using Pion :)

Ok, I will add a test

**Comment by AlexWoo** [comment]
> Great fix, thank you so much @AlexWoo
> 
> If you have time would you mind adding a test to make sure we don't break this in the future :) If not I can write it! I would move this all into a dedicated function `getFingerprint` or something like that so we can write a nice little unit test for it.
> 
> thanks so much for the contribution, and thank you for using Pion :)

I have added Test Case, and recommit in my own repo

**Comment by Sean-Der** [comment]
Merged https://github.com/pion/webrtc/commit/3a15b2ccda48f07a76f1dab1aa3f75b34f23fecc

Thank you so much @AlexWoo are you interested in contributing/being involved with Pion?

If yes I would like to add you to the organization and get your feedback on the project (what we can do better, features you would like to see) 

thank you

**Pull request closed without merging**